Automattic/sensei
# Implement getting course structure

### Changes proposed in this Pull Request

* Adds REST API endpoints for getting and saving course structure.
* _Implements_ getting course structure.
* Changes tests to allow us to use composer packages for PHP 7.2+. This was to verify JSON schema.

### Testing instructions

* Observe tests passing.
* Set up a complex course with various lessons, some in modules. Reorder the modules and lessons.
* Visit `http://{host}/wp-json/sensei-internal/v1/course-structure/{course_id}`.
* Verify JSON response is correct.
